src/worker.py

```
@@ -2845,6 +2894,440 @@ async def api_list_subjects(_req, env):
+        ).bind(survey_id).run()
+        return "ok"
+
+    result = await _survey_db(env, run)
+    if result == "not_found":
+        return err("Survey not found", 404)
+    if result == "forbidden":
+        return err("Only the survey creator can delete it", 403)
+    return ok(None, "Survey deleted")
+
+
+async def api_survey_results(req, env, survey_id: str):
```

Ruff (0.15.21)

[warning] Too many statements (64 > 50)

(PLR0915)